assets/js/modules/analytics/components/dashboard/DashboardPopularPagesWidget.js

```diff
@@ -106,7 +106,7 @@ function DashboardPopularPagesWidget() {
 
 		return [
 			title,
-			numberFormat( row.metrics[ 0 ].values[ 0 ] ),
+			numFmt( row.metrics[ 0 ].values[ 0 ], { style: 'decimal' } ),
```

Comment:
I think using `numFmt` everywhere makes sense as that's our preferred function. Since it supports everything that `numberFormat` does (and more), I think we should use it.